static/js/util/validation.js

```diff
@@ -252,12 +257,10 @@ export function validateMonth(string) {
 
 /**
  * Validate a year string is an integer and fits into YYYY
- * @param {String} string The input string
- * @returns {Number|undefined} The valid year if a valid year value or undefined if not valid
  */
-export function validateYear(string) {
+export function validateYear(string: string): ?number {
   let year = filterPositiveInt(string);
-  if (year === undefined) {
+  if (year == null) { // eslint-disable-line eqeqeq
```

Comment:
I changed the return type and I was able to revert these changes (back to just `=== undefined`)